whitepaper/dsnp_whitepaper.tex

```diff
@@ -32,25 +32,29 @@
 	\href{mailto:dsnp@projectliberty.io}{dsnp@projectliberty.io}\\
 	\url{www.projectliberty.io}
```

Comment:
Should this just be dsnp.org?

Reply:
Changing email to hello@dsnp.org. Adding dsnp.org as additional URL.